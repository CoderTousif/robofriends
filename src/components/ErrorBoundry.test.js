import React from 'react';
import ErrorBoundry from './ErrorBoundry';
import { shallow } from 'enzyme';

it('should not render ErrorBoundry component', () => {
    expect(shallow(<ErrorBoundry />)).toMatchSnapshot();
});

it('should render ErrorBoundry component', () => {
    expect(
        shallow(<ErrorBoundry />)
            .instance()
            .componentDidCatch()
    ).toMatchSnapshot();
});
