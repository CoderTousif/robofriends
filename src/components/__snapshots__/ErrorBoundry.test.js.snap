// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not render ErrorBoundry component 1`] = `ShallowWrapper {}`;

exports[`should render ErrorBoundry component 1`] = `undefined`;
